kernel_hardening_checker/checks.py

```diff
@@ -720,8 +720,8 @@ def normalize_cmdline_options(option: str, value: str) -> str:
 #    abi.vsyscall32 (any value except 2)
 #    net.ipv4.tcp_syncookies=1 (?)
 
-def add_sysctl_checks(l: List[ChecklistObjType], _arch: StrOrNone) -> None:
-# This function may be called with arch=None
+def add_sysctl_checks(l: List[ChecklistObjType], arch: str) -> None:
+    assert(arch), 'empty arch'
```

Comment:
Hmm... Please test it with `sysctl.conf`, which doesn't contain the `kernel.arch` sysctl.

Reply:
i decided to remove assert
this causes error if arch didnt detected. as we know, this line implemeted in kenel version above 6.1
near call i added soft error without sys exit
```python
if arch is None:
    print(f'[!] ERROR: {msg}')
```